opendatahub-io/opendatahub-operator
# Implement logic for Updating from Legacy Operator

Fixes #289  

## Description
This PR tests upgrade path with multi-channels

## How Has This Been Tested?
1. Test Catalog
```
quay.io/vhire/opendatahub-operator-catalog:v0.0.5
```
2. Deploy old operator on channel `rolling`
3. Create KfDef with `core` applications in `opendatahub` namespace
4. Wait for all pods to get installed
5. Switch channel to `fast`
6. Verify KfDef is deleted along with the controllers
7. Verify `default` DataScienceCluster instance is created with `none` profile
8. Enable applications that are needed

## Merge criteria:
